doc/adr/0017-system-design-lan-party.dot

```diff
@@ -0,0 +1,77 @@
+digraph system_diagram {
+    label="Lan Party System Diagram";
+
+    subgraph cluster_client {
+        label="Athens Client";
+
+        UI;
+        cSE [label="Semantic Events";];
+        "Optimistic Update";
+        cResolution [label="AGO Resolution"];
+        cTransact [label="AGO Transact"];
+        cDatascript;
+        SendIt [label="Send to Server"];
+        cAck [label="Acknowledged AGO";];
+
+        UI -> cSE;
+        cSE -> "Optimistic Update" [label="Apply optimistically";];
+        cSE -> SendIt [label="Send AGO over WS";];
+        "Optimistic Update" -> cResolution [label="Composed of AGO";];
+        cResolution -> cTransact;
+        cTransact -> cDatascript;
+
+        cAck -> cDatascript [label="Updates DB, cleaning Optimistic state";];
+        cDatascript -> UI [style=dotted; label="Render based on DB context"; color=blue];
+        cDatascript -> cResolution [style=dotted; label="Resolved in DB context"; color=blue];
+        cDatascript -> cSE [style=dotted; label="Creates AGO in DB context"; color=blue];
```

Comment:
I don't understand what this arrow means. Can you explain?

Reply:
Thinking a bit more about it, I think it means that semantic events create AGO using the DB. Is that right? If so, then I think all blue arrows are pointing in the wrong direction.

e.g. `cDatascript` doesn't use `AGO resolution` for `Resolved in DB context`, instead `AGO resolution` uses `cDatascript`

Reply:
State of Datascript DB influences resolution.

Reply:
Updated